src/lms-ds-loader/edfi_lms_ds_loader/scripts/mssql/create_assignment_submission_tables.sql

```diff
@@ -0,0 +1,72 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE TABLE lms.AssignmentSubmission (
+    AssignmentSubmissionIdentifier INT NOT NULL IDENTITY,
+    SourceSystemIdentifier NVARCHAR(255) NOT NULL,
+    SourceSystem NVARCHAR(255) NOT NULL,
+	AssignmentIdentifier INT NOT NULL,
+	LMSUserIdentifier INT NOT NULL,
+    SubmissionStatus NVARCHAR(255) NULL,
+	SubmissionDateTime DATETIME2 NULL,
+	EarnedPoints decimal NULL,
+	Grade decimal NULL,
```

Comment:
This `Grade` column data type does not match the UDM, which has varchar(20) for supporting letter grades.